src/components/Clock/Clock.tsx

```diff
@@ -1,9 +1,31 @@
-import React from 'react';
-
-import classes from './Clock.module.scss'
-
-export const Clock: React.FC = () => {
-    return (
-        <div className={classes.clock}/>
-    );
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+
+import classes from './Clock.module.scss'
+
+export const Clock: React.FC = () => {
+    const [degrees, setDegrees] = useState(calculateDegrees);
+
+    useEffect(() => {
+        const timer = setInterval(() => setDegrees(calculateDegrees()), 1000);
+        return () => clearInterval(timer);
+    }, []);
+
+    function calculateDegrees() {

```

Comment:
move function `calculateDegrees` out of component, try to keep components as small as possible